### ISSUE TITLE
How to auto start music playing when a least one client is connected? 

### ISSUE THREAD

**Issue by waldhef** [MusicPlayerDaemon/MPD#2108]
## Feature request
Hello,
I'am using `httpd` `audio_output` to stream my music in my local network.

Now I want to achieve that the music begins to play when at least on client is connected and stops playing when there are no clients connected.
I tried to set `log_level` to `verbose` in order to see something like "client connected" or "client disconnected" but there is not such a message.

Is there an other way how I can trigger `mpc play` and `mpc stop`? 

Thanks in advance.

**Comment by MaxKellermann** [comment]
No, this feature does not exist currently. But since it's open source, you can implement it (and submit a PR).

**Comment by jgregoire** [comment]
Auto-resume does work with pulseaudio outputs, including remote network outputs.

**Comment by dekiesel** [comment]
Is there any music server with that feature? 
I am trying to create a radio station for my kids for when they want to listen episodes of their favorite shows, but for that the stream needs to first start when they start their clients.
